### ISSUE TITLE
SELECT dropdown does not close when clicking outside dropdown

### ISSUE THREAD

**Issue by djmarcus1** [quantumui/quantumui#64]
Using SELECT inside a tab.

Clicking on the select shows the options properly.

Bug: clicking outside the dropdown does not close it. 

I have to click the dropdown itself to close it.

**Comment by inabhi9** [comment]
Yes, same with dropdown element. Though it's working fine on demo site

**Comment by Spharah** [comment]
I am having the same problem, i have to manually close it. Any work around?
